src/config.ts

```diff
@@ -63,9 +73,12 @@ export class Config {
     this._mongoDBUri = `mongodb://${mongoHostname}:${rawMongoPort}/${mongoDatabase}`
 
     this._featuredContentSecret = loadEnvVar('ORION_FEATURED_CONTENT_SECRET')
+    this._killSwitchSecret = loadEnvVar('ORION_KILL_SWITCH_PASSWORD')
     this._queryNodeUrl = loadEnvVar('ORION_QUERY_NODE_URL')
 
     this._isDebugging = loadEnvVar('ORION_DEBUGGING', { defaultValue: 'false' }) === 'true'
+
+    this._killerRole = 'KILLER'
```

Comment:
Can we name this role just `ADMIN`? It doesn't have to be configurable even, can be hardcoded